commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java

```diff
@@ -0,0 +1,83 @@
+/*-
+ * #%L
+ * Commons Backend - Model
+ * %%
+ * Copyright (C) 2020 - 2021 Flowing Code
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.appjars.saturn.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Utility methods for {@link Identifiable} objects. 
+ * 
+ * @author Javier Godoy / Flowing Code
+ *
+ */
+public class Identifiables {
```

Comment:
If this is a utility class, maybe it's better to call it IdentifiableUtils or IdentifiableUtilities

Reply:
In this case, `Identifiables` does with `Identifiables` the same that `java.util.Objects` does with `java.lang.Object`:
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#toString-java.lang.Object-
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hashCode-java.lang.Object-

It was non-final and had a public constructor thought (fixed in the last commit)

Reply:
Oh, I didn't know the source of the naming, it's ok, let's use that scheme.